algoliasearch-common/src/main/java/com/algolia/search/AsyncAPIClient.java

```diff
@@ -648,7 +648,11 @@ public String generateSecuredApiKey(
   }
 
   /** Package protected method for the Index class */
-  CompletableFuture<AsyncTask> deleteIndex(String indexName, RequestOptions requestOptions) {
+  CompletableFuture<AsyncTask> deleteIndex(@Nonnull String indexName) {
```

Comment:
the method should be public not protected anymore, like above

Reply:
Ok I wasn't sure 👍